ros/src/actuation/vehicles/packages/as/nodes/ssc_interface/ssc_interface.h

```diff
@@ -118,13 +119,12 @@ class SSCInterface
   void callbackFromVehicleCmd(const autoware_msgs::VehicleCmdConstPtr& msg);
   void callbackFromEngage(const std_msgs::BoolConstPtr& msg);
   void callbackFromSSCModuleStates(const automotive_navigation_msgs::ModuleStateConstPtr& msg);
-  void callbackFromSSCFeedbacks(const automotive_platform_msgs::VelocityAccelConstPtr& msg_velocity,
```

Comment:
Why was this changed?

Reply:
The velocity_accel and velocity_accel_cov essentially contain the same information. We are deprecating velocity_accel and will be switching to only using the velocity_accel_cov message, as @Daniel-Stanek mentioned to me.

Reply:
Have you verified with Dan that this is what we are using on any vehicles with a _released_ version of SSC?

Reply:
It looks like 2.1.1 release of `ros_vehicle` (which means all ssc_ packages) provide this. I'm good with it.